UnoCheck/Manifest/Gtk.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace DotNetCheck.Manifest
+{
+	public partial class Gtk
+	{
+		[JsonProperty("urls")]
+		public Urls Urls { get; set; }
+
+		[JsonIgnore]
+		public System.Uri Url
+			=> Urls?.Get(CompatVersion);
+
+		[JsonProperty("version")]
+		public string Version { get; set; }
+
+		[JsonProperty("minimumVersion")]
+		public string MinimumVersion { get; set; }
+
+		[JsonIgnore]
+		public string CompatVersion
+			=> Version ?? MinimumVersion;
+
+		[JsonProperty("requireExact")]
+		public bool RequireExact { get; set; }
```

Comment:
Are these needed at this point? (Maybe by the `Get` method?)

Reply:
I removed the RequireExtac because is not used. I don't know if I got your point, please let me know any change.